test/e2e/adminUI/pages/home.js

```diff
@@ -1,95 +1,38 @@
+var dashboardGroup = require('./dashboardGroup');
+
+var accessdashboardGroup = dashboardGroup({
+	groupName: 'Access',
+	tabs: [
+		{name: 'users', items: '1 Item'},
```

Comment:
why need to specify the item count here?  what if the test changes it?  for sure I missed something :-)